mclazy.py

```diff
@@ -200,16 +201,19 @@ def main():
 
     # read defaults from command line arguments
     parser = argparse.ArgumentParser(description='Automatically build Fedora packages for a GNOME release')
-    parser.add_argument('--fedora-branch', default="rawhide", help='The fedora release to target (default: rawhide)')
+    parser.add_argument('fedora_branch', metavar='BRANCH', help='The fedora release to target')
+    parser.add_argument('--allow-eol', action='store_true', help='Allow operation on EOL branches')
     parser.add_argument('--simulate', action='store_true', help='Do not push any changes')
     parser.add_argument('--no-simulate', action='store_true', help='Push changes')
     parser.add_argument('--check-installed', action='store_true', help='Check installed version against built version')
     parser.add_argument('--relax-version-checks', action='store_true', help='Relax checks on the version numbering')
     parser.add_argument('--no-build', action='store_true', help='Do not actually build, e.g. for rawhide')
     parser.add_argument('--no-mockbuild', action='store_true', help='Do not do a local mock build')
-    parser.add_argument('--no-rawhide-sync', action='store_true', help='Do not push the same changes to git rawhide branch')
+    parser.add_argument('--rawhide-sync', default=None, action='store_true', help='Push the same changes to git rawhide branch. Used by default when appropriate')
+    parser.add_argument('--no-rawhide-sync', action='store_false', dest="rawhide_sync", help='Disable --rawhide-sync')
```

Comment:
Can we get rid of `--rawhide-sync`, then? I think users will never want to do this except when specified by the config.